### ISSUE TITLE
Accuracy=0 ,Loss=0, Perplexity = 1 during evaluate

### ISSUE THREAD

**Issue by ericosmic** [tensorflow/magenta#1319]
When I run the evaluate progress while training,  the eval result always staying at ''Accuracy=0 ,Loss=0, Perplexity = 1 " ,   who know what's wrong with this .  
I use cuda9+cudnn7+tf1.11 .

**Comment by adarob** [comment]
which model?

**Comment by ericosmic** [comment]
@adarob   melody_rnn

**Comment by eps696** [comment]
observed the same thing with newly trained performance_rnn model (run eval once after the training). 
there were other problems with that process though, possibly unrelated -  https://github.com/tensorflow/magenta/issues/1364

**Comment by NishthaN** [comment]
I'm facing the same issue. Has there been any solution found to this?

**Comment by adarob** [comment]
Is it possible your training set is empty? How many examples are you training on?

**Comment by NishthaN** [comment]
The training set wasn't empty. The training of the model was done successfully. Am using this MIDI dataset here: https://mega.nz/#!Elg1TA7T!MXEZPzq9s9YObiUcMCoNQJmCbawZqzAkHzY4Ym6Gs_Q. I converted these MIDI files in NodeSequences and then SequenceExamples as per the tutorial given for Melody RNN.
